### DISCUSSION TITLE
Installation instructions are terrible

### DISCUSSION THREAD

**Discussion by Hank3000** [Keriew/augustus#479] [General]
I've tried several times to install Augustus. 

The instructions in the README file are useless.

Why would you make time to create something like this and not then make time to show people how to actually play it?

**Comment by crudelios** [comment]
Basic installation is quite simple, as stated in the readme:

- Get the original C3
- Download Augustus
- Place the entire contents of the augustus zip on your C3 folder
- Run augustus.exe

If this doesn't work, then you have an extra issue, most likely related to having duplicate C3 or augustus folders already.

To further help you, we need to know what's wrong in your case. We'll be more than happy to help you!

**Reply by Hank3000** [reply]
There is no augustus.exe 

so not quite so simple

**Reply by crudelios** [reply]
The download is a zip file which you need to extract and augustus.exe is inside. If you use Windows, double-clicking the downloaded zip file will open it and show all the files (including augustus.exe), which you can then copy over to the C3 directory.

**Reply by Hank3000** [reply]
Yes I downloaded it (augustus-3.0.1). 
Extracted it. 
Coped the contents to the existing C3 folder installed by steam.
And there is no augustus.exe

**Reply by crudelios** [reply]
Can you please recheck the zip file? Or perhaps even download it again? I just downloaded it to check and `augustus.exe` is definitely in there.  There should be an `assets`, `maps` and `manual` folders, 3 `dll` files and `augustus.exe` inside.

Just to be sure, here's the direct link: 

https://augustus.josecadete.net/download/augustus-3.0.1-windows.zip

**Reply by Hank3000** [reply]
I have checked it several times and used windows to search the folder for augustus.exe and there is no augustus.exe

**Reply by Hank3000** [reply]
OK I have downloaded the one via the link you provided, which does actually contain augustus.exe.

I have copied all files again, including the assets folder. Now I get the message "can't find assets folder".

So I manually coped just the assets folder from the new folder created from the file I downloaded via your link. I launched it and I still get the message "can't find assets folder".

So I searched the forums which has a thread within which there is a link to just downloaded the assets folder. I extracted it and coped all its contents. I launched it and I still get the message "can't find assets folder".

So I completely deleted the existing assets folder and replaced it with the two different assets folders I have downloaded, the one from the link you provided and the one which is a download of just the assets folder. Each time I launched it I still get the message "can't find assets folder".

**Reply by crudelios** [reply]
All right, at least there's progress. Can you post the contents of `augustus-log.txt` here, please?

**Reply by Hank3000** [reply]
INFO: Augustus version 3.0.1
INFO: Initializing SDL
INFO: SDL initialized
INFO: Loading game from working directory
ERROR: 'c3.eng' or 'c3_mm.eng' files not found or too large.
INFO: Loading game from base path C:\Program Files (x86)\Steam\steamapps\common\Caesar 3\
ERROR: 'c3.eng' or 'c3_mm.eng' files not found or too large.
INFO: Location:  C:\Users\Hank3000\AppData\Roaming\augustus\augustus\
INFO: Loading game from user pref C:\Program Files (x86)\Steam\steamapps\common\Caesar 3\C3
INFO: Config key  enable_audio  1
INFO: Config key  master_volume  100
INFO: Config key  ui_walker_waypoints
INFO: Config key  gameplay_fix_immigration
INFO: Config key  gameplay_fix_100y_ghosts
INFO: Config key  screen_display_scale  100
INFO: Config key  screen_cursor_scale  100
INFO: Config key  screen_color_cursors
INFO: Config key  ui_sidebar_info  1
INFO: Config key  ui_show_intro_video
INFO: Config key  ui_smooth_scrolling  1
INFO: Config key  ui_disable_mouse_edge_scrolling
INFO: Config key  ui_visual_feedback_on_delete
INFO: Config key  ui_allow_cycling_temples
INFO: Config key  ui_show_water_structure_range  1
INFO: Config key  ui_show_construction_size  1
INFO: Config key  ui_zoom  1
INFO: Config key  ui_highlight_legions  1
INFO: Config key  ui_show_military_sidebar
INFO: Config key  ui_disable_map_drag
INFO: Config key  ui_show_max_prosperity
INFO: Config key  gameplay_change_jealous_gods
INFO: Config key  gameplay_change_global_labour
INFO: Config key  gameplay_change_retire_at_60
INFO: Config key  gameplay_change_fixed_workers
INFO: Config key  gameplay_wolves_block
INFO: Config key  gameplay_buyers_dont_distribute
INFO: Config key  gameplay_change_getting_granaries_go_offroad
INFO: Config key  gameplay_change_granaries_get_double
INFO: Config key  gameplay_change_allow_exporting_from_granaries
INFO: Config key  gameplay_change_tower_sentries_go_offroad
INFO: Config key  gameplay_change_farms_deliver_close
INFO: Config key  gameplay_change_only_deliver_to_accepting_granaries
INFO: Config key  gameplay_change_all_houses_merge
INFO: Config key  gameplay_change_random_mine_or_pit_collapses_take_money
INFO: Config key  gameplay_change_multiple_barracks
INFO: Config key  gameplay_change_warehouses_dont_accept
INFO: Config key  gameplay_change_houses_dont_expand_into_gardens
INFO: Config key  gameplay_change_monuments_boost_culture_rating
INFO: Config key  gameplay_change_disable_infinite_wolves_spawning
INFO: Config key  gameplay_change_romers_dont_skip_corners
INFO: Config key  ui_language_dir
INFO: Config value  
INFO: Detected language:  English
INFO: Detected encoding:  1252
INFO: Creating screen 1920 x 1080, fullscreen, driver: windows
INFO: Creating renderer
INFO: Textures created (1920 x 1080)
INFO: Model loaded
INFO: Asset path detected at  ./assets
INFO: Loading assetlist file  aesthetics.xml
ERROR: Error parsing file  aesthetics.xml
INFO: Loading assetlist file  building_upgrades.xml
ERROR: Error parsing file  building_upgrades.xml
INFO: Loading assetlist file  circus.xml
ERROR: Error parsing file  circus.xml
INFO: Loading assetlist file  colosseum.xml
ERROR: Error parsing file  colosseum.xml
INFO: Loading assetlist file  construction_guilds.xml
ERROR: Error parsing file  construction_guilds.xml
INFO: Loading assetlist file  econ_logistics.xml
ERROR: Error parsing file  econ_logistics.xml
INFO: Loading assetlist file  entertainment.xml
ERROR: Error parsing file  entertainment.xml
INFO: Loading assetlist file  grand_temples.xml
ERROR: Error parsing file  grand_temples.xml
INFO: Loading assetlist file  military_buildings.xml
ERROR: Error parsing file  military_buildings.xml
INFO: Loading assetlist file  minor_monuments.xml
ERROR: Error parsing file  minor_monuments.xml
INFO: Loading assetlist file  roadblocks.xml
ERROR: Error parsing file  roadblocks.xml
INFO: Loading assetlist file  ui_elements.xml
ERROR: Error parsing file  ui_elements.xml
INFO: Loading audio files
INFO: Window resized to 1920 x 1080
INFO: Textures created (1920 x 1080)
INFO: Window 1 shown
INFO: Loading saved game  mission1.pak
INFO: Savegame version  102
INFO: Saving game  Citizen.svx
INFO: Exiting game

**Reply by crudelios** [reply]
Ok, so the game is detecting an `assets` folder but its contents are wrong. This happens because you probably have the beta assets from the topic you viewed.

Please go to the folder `C:\Program Files (x86)\Steam\steamapps\common\Caesar 3\C3`, delete the `assets` folder there and then copy the one from the link I posted above.

If it works please just let me know, otherwise post the log file again.

Also, we'll be releasing v3.1 soon which will (hopefully) fix this assets issue.

**Reply by Hank3000** [reply]
I already did that. I said I did that in my earlier message. But I will do it again.

**Reply by crudelios** [reply]
Please do, as the game is indeed fetching the assets for the newer beta version and not the ones it's supposed to.
